assets/js/blocks/checkout/inner-blocks/checkout-order-note-block/style.scss

```diff
@@ -17,6 +17,6 @@
 	margin-top: $gap;
 }
 
-.wc-block-checkout__order-notes.wc-block-components-checkout-step {
+.wc-block-components-form .wc-block-checkout__order-notes.wc-block-components-checkout-step {
```

Comment:
It was required to increase the specific of this selector; otherwise it was being overridden by another one.